style.css

```diff
@@ -0,0 +1,165 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins:200i,400&display=swap');
+:root {
+    --color-white: #f3f3f3;
+    --color-darkblue: #4d588a;
+    --color-darkblue-alpha: rgba(27, 27, 50, 0.8);
+    --color-green: #37af65;
+}
+
+*,
+*::before,
+*::after {
+    box-sizing: border-box;
+}
+
+body {
+    font-family: 'Poppins', sans-serif;
+    font-size: 1rem;
+    font-weight: 400;
+    line-height: 1.4;
+    color: var(--color-white);
+    margin: 0;
+}
+
+body::before {
+    content: '';
```

Comment:
Avoid leaving empty properties I suggest you fill in the property or you remove it to make your code clean.

Reply:
KIndly make sure the request you are asking does not look like an order 😄 

you could say:

your code would look cleaner if you avoid leaving empty properties, kindly remove empty properties from your code

or 

please avoid leaving unused properties, I would suggest you fill the property or remove it so that your code looks cleaner.(The "please" makes the tone appear less like an order)